docker/deployed/staging/healthbot/config.json

```diff
@@ -15,6 +15,10 @@
                 "CheckInterval": "360s",
                 "ReceiptTimeout": "20s",
                 "Tablename": "${HEALTHBOT_OPTIMISM_GOERLI_TABLE}"
+            },
```

Comment:
Set this only for `staging`

Reply:
SGTM. Just in case, did you try doing a `[testnetdeploy]` in this branch to double-check not setting the new `OverrideClient` works fine? Just to be sure it will do the correct thing in mainnet too.

Reply:
Will do that